src/threadpool.c

```diff
@@ -224,20 +243,8 @@ static void init_threads(void) {
   uv__queue_init(&slow_io_pending_wq);
   uv__queue_init(&run_slow_work_message);
 
-  if (uv_sem_init(&sem, 0))
-    abort();
-
-  config.flags = UV_THREAD_HAS_STACK_SIZE;
-  config.stack_size = 8u << 20;  /* 8 MB */
-
-  for (i = 0; i < nthreads; i++)
-    if (uv_thread_create_ex(threads + i, &config, worker, &sem))
-      abort();
-
-  for (i = 0; i < nthreads; i++)
-    uv_sem_wait(&sem);
-
-  uv_sem_destroy(&sem);
+  /* Spin up the default number of threads */
+  spin_threads(0 /* from */, nthreads /* to */);
```

Comment:
This call is fine because it's inside `init_threads`, which is guarateed to only run once. Any other place where this is called needs to hold onto the mutex.